packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,1933 @@
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ResourcePost'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/ResourcePost'
+      responses:
+        '201':
+          description: New draft resource
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    type: integer
+                    default: 12
+                  estimatedTotal:
+                    $ref: '#/components/schemas/MetaCount'
+                  resourceIssues:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        _id:
+                          type: string
+                          default: '5f7a462dcd1b786bed19b4cb'
+                        detail:
+                          type: object
+                          properties:
+                            kind:
+                              $ref: '#/components/schemas/MetaEventKindUserReport'
+                        kind:
+                          $ref: '#/components/schemas/MetaEventKindUserReport'
+                        resolved:
+                          type: boolean
+                          default: false
+                        resourceId:
+                          type: string
+                          default: '5def5cb7b831a048f2a3deed'
+                        severity:
+                          type: string
+                          default: 'high'
+                        createdAt:
+                          $ref: '#/components/schemas/MetaDateTime'
+                        lastModifiedAt:
+                          $ref: '#/components/schemas/MetaDateTime'
+                        __v:
+                          type: integer
+                          default: 0
+        '401':
+          $ref: '#/components/responses/ErrorAuthentication'
+        '422':
+          $ref: '#/components/responses/ErrorAlreadyExists'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resource/{id}:
+    parameters:
+      - $ref: '#/components/parameters/idPath'
+    get:
+      description: 'Get a resource.'
+      summary: 'Get a resource'
+      tags:
+        - resource
+      operationId: resourceGet
+      responses:
+        '200':
+          description: Single Resource.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  resource:
+                    $ref: '#/components/schemas/Resource'
+        '404':
+          $ref: '#/components/responses/ErrorNotFound'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resources:
+    parameters:
+      - $ref: '#/components/parameters/idQueryOptional'
```

Comment:
This can actually be [one or many, comma separated IDs](https://github.com/CodeForBoulder/upswyng/blob/master/packages/server/src/routes/api/resources/index.ts#L7). Doesn't reflect quite like that with this parameter.

Reply:
updated to reference the support for multiple id's in the description